recipes/libexif/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+import contextlib
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class LibexifConan(ConanFile):
+    name = "libexif"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://libexif.github.io/"
+    license = "LGPL-2.1"
+    description = "libexif is a library for parsing, editing, and saving EXIF data."
+    topics = ("exif", "metadata", "parse", "edit")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
+    @contextlib.contextmanager
+    def _build_context(self):
+        if self.settings.compiler == "Visual Studio":
+            with tools.vcvars(self):
+                env = {
+                    "CC": "cl -nologo",
+                    "LD": "link -nologo",
+                }
+                with tools.environment_append(env):
+                    yield
+        else:
+            yield
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def build_requirements(self):
+        self.build_requires("gettext/0.21")
+        self.build_requires("libtool/2.4.6")
+        if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
+            self.build_requires("msys2/cci.latest")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self, win_bash=self._settings_build.os == "Windows")
```

Comment:
Try this when the build fails.
When it keeps failing, I'll take a look.
```suggestion
        self._autotools = AutoToolsBuildEnvironment(self, win_bash=self._settings_build.os == "Windows")
        self._autotools.libs = []
```

Reply:
Try my suggestion :arrow_up: .
If that fails, I'll dive in.